std/typecons.d

```diff
@@ -5121,9 +5121,28 @@ enum RefCountedAutoInitialize
 
 /**
 Defines a reference-counted object containing a $(D T) value as
-payload. $(D RefCounted) keeps track of all references of an object,
-and when the reference count goes down to zero, frees the underlying
-store. $(D RefCounted) uses $(D malloc) and $(D free) for operation.
+payload.
+
+$(D RefCounted) is implemented as a pointer to a data structure, which
+is refered to as the $(I store) or $(I storage implentation struct) in
```

Comment:
Nitpick: spelling "refe**rr**ed" (two r's in the middle), and "imple**me**ntation".